generator/templates/enums/template_numeric.h

```diff
@@ -0,0 +1,11 @@
+{% extends "template.h" %}
```

Comment:
As part of refusing from any kind of customisation, I suggest to delete generator/templates/enums/template_numeric.h and do not process string and number enums separately? Otherwise how to evaluate what enum should use template_numeric? (E.g. on example of LightName)